pwned_pass.sh

```diff
@@ -29,7 +29,12 @@ if [ -z "$password" ]; then
 fi
 set -u
 
-hash=$(printf "%s" "$password" | openssl sha1 | awk '{print $2}')
+if [ $(uname) == "Linux" ]; then
+    hash=$(printf "%s" "$password" | openssl sha1 | awk '{print $2}')
+elif [ $(uname) == "Darwin" ]; then
+    hash=$(printf "%s" "$password" | openssl sha1 | awk '{print $1}')
```

Comment:
This breaks compatibility with folks that have openssl from ports installed. I'd recommend instead just taking the last word:

```
 hash=$(printf "%s" "$password" | openssl sha1 | awk '{print $NF}') 
```

This also eliminates the need to conditionalize on uname.